common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -27,7 +27,21 @@ run() {
 
 killall_program() {
     PROGRAM_NAME=$1
-    KILL_SIG=${2:-TERM}
+    KILL_SIG=${2:-KILL}
+    TERM_SIG=${2:-TERM}
+    TIMEOUT=10
+    echo "terminating $PROGRAM_NAME ($TERM_SIG)"
+    start-stop-daemon -K -s "$TERM_SIG" -x "$PRPLMESH_BIN_DIR"/"$PROGRAM_NAME" > /dev/null 2>&1
+
+    # `-R` option is not always awailable for `start-stop-daemon`, we need to imitate it
+    for i in $(seq 1 "$TIMEOUT"); do
+        start-stop-daemon -T -x "$PRPLMESH_BIN_DIR"/"$PROGRAM_NAME" > /dev/null 2>&1
+        if $?; then
+           return
+        fi
```

Comment:
```suggestion
        if ! start-stop-daemon -T -x "$PRPLMESH_BIN_DIR"/"$PROGRAM_NAME"; then
           return
        fi
        sleep 1
```

There were two things very wrong here:
- start-stop-daemon -T returns true if the process is running; we want to return when it is *not* running.
- `if $?` will try to run the program `0` or `1`, neither of which exists, so you'll get an error message like `0: command not found...` and it will never return.

Also, start-stop-daemon -T normally doesn't produce any output. If it does, it's some kind of error message, and we probably want to see it. So redirecting is not needed and probably a bad idea.

Finally, we should leave a bit of time between iterations.